daemon/albatross_console.ml

```diff
@@ -36,14 +44,6 @@ let read_console id name ring fd =
            Vmm_lwt.write_wire fd (header, data) >>= function
            | Error _ ->
              Vmm_lwt.safe_close fd >|= fun () ->
```

Comment:
here inside of `f`, we have shadowed the outer `fd`, and what we update below needs to be the inner one.

so, the outer one is the one we read from (i.e. the unikernel output), while the one in `f` is the one from the client, where we write to (and which we need to close on exception / error).

Reply:
Oh, you are right! I missed that it was overshadowed.